src/en/authors-charm-with-docker.md

```diff
@@ -0,0 +1,325 @@
+# Charming with Docker
+
+You have a Docker container and you heard about Juju.  Juju can deploy your
```

Comment:
We should probably remove this line.  If they are in the Juju documentation they know about Juju.